manifests/virt-handler.yaml.in

```diff
@@ -29,6 +30,8 @@ spec:
         volumeMounts:
         - name: libvirt-runtime
           mountPath: /var/run/libvirt
+        - name: sockets
```

Comment:
It's possible in the future that there will be more data related to a domain that needs to be shared between local KubeVirt components than just the unix file.  For example, the cloud-init nocloud data source is going to require us sharing a .iso file between virt-handler and libvirt. 

maybe we should consider making this a generic share folder instead of labeling it as "sockets".  Maybe something like "domain-share" or something like that?

Reply:
yeah, I agree, lets not derail this pr with a discussion about a generic shared directory.   We can pick up talking about this another time. IMO, The current approach in the pr is fine for now.